### DISCUSSION TITLE
Mapping localleader

### DISCUSSION THREAD

**Discussion by fikovnik** [LazyVim/LazyVim#260] [General]
I was wondering how to properly map keys with `<localleader>`?

I have the following in my `options.lua`:

```lua
vim.g.maplocalleader = "m"
```

In `plugins/lsp.lua` I define:

```lua
return {
  {
    "neovim/nvim-lspconfig",
    init = function()
      local keys = require("lazyvim.plugins.lsp.keymaps").get()

      keys[#keys + 1] = { "gd", false }
      keys[#keys + 1] = { "<localleader>d", "<cmd>Telescope lsp_definitions<cr>", desc = "Definitions" }
    end,
  },
}
```

But, when I open an lsp supported source file, the `md` mapping does not work.
If I run `:lua require("which-key").show("m", {mode = "n", auto = true})`, it shows the mapping and after that it is working.
btw: If I hit the `m` key while the lsp is loading, the progress of loading the lsp (using fidget for that) stops until I hit other key.

**Comment by folke** [comment]
What does `:verbose nmap md` say?

Loading progress stops, probably because `m` is bound to something that expects input.

`m` is the keybinding for marks, so maybe that is interfering somehow.

**Reply by fikovnik** [reply]
```
:verbose nmap md
n  md          *@<Cmd>Telescope lsp_definitions<CR>
                 Definitions
        Last set from Lua
```

In my old diy-config with which-key it works well. However, in that config when I do `:nmap m` it does show the call to `which-key`:

```
n  m           * <Cmd>lua require("which-key").show("m", {mode = "n", auto = true})<CR>
```

While in my new LazyVim-based it does not.
